### DISCUSSION TITLE
Permission issue?

### DISCUSSION THREAD

**Discussion by SteveIreland** [opensourcepos/opensourcepos#3665] [Team Posts]
Just checking to see if I've been decommissioned.

I don't seem to be able to do anything with the ospos site.

**Comment by daN4cat** [comment]
what do you mean? The demo web site or something else?

**Comment by SteveIreland** [comment]
You can ignore this too.  The github crash overnight just messed with my mind.  Everything I updated, posted, even THIS thread disappeared as soon as I posted it.  I had no idea it was actually accepted it since to me it looked like it was never accepted.
